stable/redmine/values.yaml

```diff
@@ -178,3 +178,23 @@ persistence:
   # storageClass: "-"
   accessMode: ReadWriteOnce
   size: 8Gi
+
+## Define a disruption budget
+## ref: https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
+podDisruptionBudget:
+  enabled: false
+  # minAvailable: 1
+  # maxUnavailable: 1
+
+## Define the number of pods the deployment will create
+## ref: https://kubernetes.io/docs/concepts/workloads/controllers/deployment/
+replicas: 1
```

Comment:
I have some concerns about the replicas configuration... Unless you use some NFS Volume Provisioner, you shouldn't use more than 1 replica... 

There's an interesting article about the requirements to scale horizontally this kind of applications:
https://engineering.bitnami.com/articles/scaling-wordpress-in-kubernetes.html

It's oriented to WP but the same concepts apply to the Redmine chart.

Reply:
I agree, you should only increase replicas if you're using network attached storage, like NFS. I could add such a warning to the values file. Or would you prefer a different solution?

Reply:
I think we should add a warning both on the **values.yaml** and the **README.md** file.